docs/events/timeline-templates.asciidoc

```diff
@@ -90,6 +88,10 @@ image::images/create-a-timeline-filter.png[Shows an example of a Timeline filter
 TIP: You can also drag and send items to the template from the *Overview*,
 *Hosts*, *Network*, and *Alerts* pages.
 
++
+
```

Comment:
The plus is rendering in the preview, and the next bullet goes back to "1". Maybe the whitespace needs to be removed

Reply:
yeah the extra space is throwing the numbering off. Thanks for catching this!

Reply:
Fixed in [c88e8a6](https://github.com/elastic/security-docs/pull/4299/commits/c88e8a6b2e2e9797f17ac9dd11487e4a1a1cd946).